person.rb

```diff
@@ -1,14 +1,21 @@
+require './nameable'
 attr_accessor :name, :age
+
 attr_reader :id
 
-class Person
+class Person < nameable
```

Comment:
- [ ] Also, please correct the casing of the parent class name. It should be `Nameable` instead. 🤓 🔧